lib/services/metadata_provider.dart

```diff
@@ -49,182 +22,200 @@ class MetadataProvider {
   LyricDto? lyrics;
   bool isDownloaded;
   bool qualifiesForPlaybackSpeedControl;
+  double? parentNormalizationGain;
```

Comment:
This can be "BaseItemDto? parentItem` so that this change is generic to support other cases. Or is it better to leave as is?

Reply:
not really. a parent item can also be a playlist, if playback was started via that playlist...